plugins/Observability/src/test/report-generator-for-regression-testing.ssc

```diff
@@ -0,0 +1,63 @@
+// Name: Observability Plugin Report Generator For Regression Testing
+// License: Public Domain
+// Author: colossatr0n
+// Version: 1.0
+// Description: Generates reports to be used for regression testing. Run this script in two different versions of Stellarium and compare the output using some kind of differ (vim, git, diff). For macOS, file is output to "~/Library/Application Support/Stellarium/".
+
+const objectNames = [
+    "Sun",      // Sun
+    "Moon",     // Moon
+    "Jupiter",  // Solar system planet
+    "Achernar", // Interstellar object
```

Comment:
Actually very much a "Stellar" object, i.e. star ;-)

Reply:
Oh, sorry. Will change that.